### PR TITLE
Update Endpoints samples:

### PR DESCRIPTION
**Pull request by sepehre**
1. Change default service names to the new cloud.goog format.
2. Change app.yaml beta_settings to endpoints_api_service.

### CODE REVIEW

**Comment by sepehre** [comment]
Please do not merge until documentation updated have been coordinated.

File: endpoints/getting-started/src/main/appengine/openapi.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ info:
   description: "A simple Google Cloud Endpoints API example."
   title: "Endpoints Example"
   version: "1.0.0"
-host: "YOUR-PROJECT-ID.appspot.com"
+host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"
```
Comment (lesv) [inline endpoints/getting-started/src/main/appengine/openapi.yaml]: Can you use https with this?

Comment (sepehre) [inline endpoints/getting-started/src/main/appengine/openapi.yaml]: This field is not a DNS name that the application will be called at (not yet at least). It is the Endpoints service name.

Comment (Eng-Fouad) [inline endpoints/getting-started/src/main/appengine/openapi.yaml]: I wonder why you chose `cloud.goog` :)

**Comment by lesv** [comment]
You've got some conflicting changes in `endpoints/getting-started/src/main/appengine/openapi.yaml`.

Also, any chance you could update to the new maven plugin?  https://cloud.google.com/appengine/docs/java/tools/using-maven  We'll have to do it early Jan.  The commands become appengine:run & appengine:deploy

**Comment by sepehre** [comment]
Let's abandon this PR in favor of https://github.com/GoogleCloudPlatform/java-docs-samples/pull/447

If you don't mind, let's please do the maven plugin change separately

**Comment by lesv** [comment]
Sg.

**Pull request closed without merging**